server/tests/factories.js

```diff
@@ -0,0 +1,74 @@
+import Faker from 'faker';
+import User from '../models/user';
+import Device from '../models/device';
+import Group from '../models/group';
+import Ticket from '../models/ticket';
+import EscalationPolicy from '../models/escalationPolicy';
+
+const fixtures = {
+  user(user = {}) {
+    return {
+      id: user.id,
+      name: user.name || Faker.internet.userName(),
+      email: user.email || Faker.internet.email()
+    };
+  },
+  email_device(device = {}) {
+    return {
+      name: device.name || Faker.lorem.word(),
+      type: device.type || 'email',
+      contactInformation: device.contactInformation || Faker.internet.email()
+    };
+  },
+  sms_device(device = {}) {
+    return {
+      name: device.name || Faker.lorem.word(),
+      type: device.type || 'sms',
+      contactInformation: device.contactInformation || Faker.phone.phoneNumber()
+    };
+  },
+  group(group = {}) {
+    return {
+      name: group.name || Faker.internet.userName(),
+      users: group.users || ['123456789012345678901234']
```

Comment:
Can you make this a constant and also export it so that you can reference it in a test?